frontend/src/metabase/query_builder/components/view/View.jsx

```diff
@@ -204,22 +210,19 @@ class View extends Component {
     const isNewQuestion = !isNative && Lib.sourceTableOrCardId(query) === null;
 
     return (
-      <Motion
-        defaultStyle={isNewQuestion ? { opacity: 0 } : { opacity: 1 }}
-        style={isNewQuestion ? { opacity: spring(0) } : { opacity: spring(1) }}
-      >
-        {({ opacity }) => (
-          <QueryBuilderViewHeaderContainer>
-            <BorderedViewTitleHeader {...this.props} style={{ opacity }} />
-            {opacity < 1 && (
-              <NewQuestionHeader
-                className="spread"
-                style={{ opacity: 1 - opacity }}
-              />
-            )}
-          </QueryBuilderViewHeaderContainer>
-        )}
-      </Motion>
+      <QueryBuilderViewHeaderContainer>
+        <BorderedViewTitleHeader
+          {...this.props}
+          style={{
+            transition: "opacity 300ms linear",
+            opacity: isNewQuestion ? 0 : 1,
+          }}
+        />
```

Comment:
```suggestion
     <Transition keepMounted mounted={isNewQuestion} transition={fadeIn} duration={300}>
        <BorderedViewTitleHeader {...this.props} />
    </Transition>
```
Just curious, would something like this work?

Reply:
I gave it a quick try. `<BorderedViewTitleHeader />` didn't like being put in the transition layout wise, though that's probably fixable. The transition bit worked just fine though